mobile/lib/pages/plyta_tygodnia_page.dart

```diff
@@ -0,0 +1,232 @@
+  static const EdgeInsets _pagePadding =
+      EdgeInsets.only(top: 26, left: 26, right: 26);
+
+  /// Space between things on the page.
+  static const SizedBox _emptySpace = SizedBox(height: 9);
+
+  /// Plyta tygodnia info fetch details.
+  static final Uri _infoUrl = Uri.parse(
+    'https://radioaktywne.pl/wp-json/wp/v2/album?page=1&per_page=16',
+  );
+  static const _infoHeaders = {'Content-Type': 'application/json'};
+
+  /// Plyta tygodnia album cover fetch details.
+  static Uri _imgUrl(String id) => Uri.parse(
+        'https://radioaktywne.pl/wp-json/wp/v2/media?include[]=$id',
+      );
+  static const _imgHeaders = {'Content-Type': 'image/jpeg'};
+
+  /// Fetch current Plyta tygodnia from radioaktywne.pl api.
+  Future<PlytaTygodniaInfo> _fetchPlytaTygodnia() async {
+    try {
+      final data = await fetchData(
+        _infoUrl,
+        PlytaTygodniaInfo.fromJson,
+        headers: _infoHeaders,
+        timeout: timeout,
+      );
+
+      final plytaTygodnia = data.first;
+
+      final imageUrls = await fetchData(
+        _imgUrl(plytaTygodnia.imageTag),
+        (e) => (e['guid'] as Map<String, dynamic>)['rendered'] as String,
+        headers: _imgHeaders,
+        timeout: timeout,
+      );
+
+      final imageUrl = imageUrls.first;
+      plytaTygodnia.imageTag = imageUrl;
+
+      return plytaTygodnia;
+    } on TimeoutException catch (_) {
+      return PlytaTygodniaInfo.empty();
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return RefreshableFetchWidget(
+      onFetch: _fetchPlytaTygodnia,
+      defaultData: PlytaTygodniaInfo.empty(),
+      loadingBuilder: (context, snapshot) => const _PlytaTygodniaWaiting(),
+      errorBuilder: (context) => const _PlytaTygodniaNoData(),
+      builder: (context, plytaTygodnia) => Padding(
+        padding: _pagePadding,
+        child: ListView(
+          children: [
+            AspectRatio(
+              aspectRatio: 1,
+              child: Image.network(
+                plytaTygodnia.imageTag,
+                loadingBuilder: (context, child, loadingProgress) =>
+                    loadingProgress == null
+                        ? child
+                        : Container(
+                            color: context.colors.backgroundDarkSecondary,
+                            child: Center(
+                              child: SizedBox(
+                                width: 40,
+                                height: 40,
+                                child: CircularProgressIndicator(
+                                  color: context.colors.highlightGreen,
+                                  strokeWidth: 5,
+                                ),
+                              ),
+                            ),
+                          ),
+                errorBuilder: (context, child, loadingProgress) => Center(
+                  child: Text(
+                    'Wystąpił błąd podczas pobierania obrazu',
```

Comment:
ohhh nooo, why are you not using https://github.com/RadioAktywne/ra-app/blob/master/mobile/lib/l10n/app_pl.arb

Reply:
we can connect and I'll show you what's up

Reply:
Good tip, just added that